### PR TITLE
Update couple links to test-infra home page

### PR DESCRIPTION
**Pull request by krzyzacy**
People are asking, and I'd rather not dig into our history next time :-)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,6 +36,11 @@ collects test results from the last 24 hours. It is updated hourly by the
 scripts under `jenkins/test-history`.
 * [Gubernator](https://k8s-gubernator.appspot.com/) is a Google App Engine site
 that parses and presents the results from individual test jobs.
+* [Prow](http://prow.k8s.io) is the system that handles GitHub events and commands for Kubernetes.
```
Comment (ixdy) [inline README.md]: https?

Comment (krzyzacy) [inline README.md]: done.

seems it's default to http if I don't specify https

**Comment by ixdy** [comment]
@k8s-ci-robot nobody likes you

**Comment by krzyzacy** [comment]
poor @k8s-ci-robot

**Pull request merged**